app/javascript/dashboard/i18n/locale/en/sla.json

```diff
@@ -0,0 +1,63 @@
+{
+  "SLA": {
+    "HEADER": "SLA",
+    "HEADER_BTN_TXT": "Add SLA",
+    "LOADING": "Fetching SLAs",
+    "SEARCH_404": "There are no items matching this query",
+    "SIDEBAR_TXT": "<p><b>SLA</b> <p>Think of Service Level Agreements (SLAs) like friendly promises between a service provider and a customer.</p> <p> These promises set clear expectations for things like how quickly the team will respond to issues, making sure you always get a reliable and top-notch experience!</p>",
+    "LIST": {
+      "404": "There are no SLAs available in this account.",
+      "TITLE": "Manage SLA",
+      "DESC": "SLAs: Friendly promises for great service!",
+      "TABLE_HEADER": ["Name", "Description", "FRT", "NRT", "RT", "Business Hours"]
```

Comment:
We could combine name description and business hours. Can you work with @absurdiya for an updated design?